plugins/ptp_operator/ptp_operator_plugin.go

```diff
@@ -607,7 +628,7 @@ func processMessages(c net.Conn) {
 	}
 }
 
-// HasEqualInterface checks if configmap  changes has same interface
+// HasEqualInterface checks if configmap changes has same interface
```

Comment:
If you're changing the comment, can you make it more clear?  I'm not really sure what it's doing from this comment.